### PR TITLE
More changes and documentation

### PR DESCRIPTION
**Pull request by thibault-martinez**
## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: documentation/docs/02_account_manager.md [hunk 1]
```diff
@@ -6,59 +9,93 @@ The account manager interface allows you to:
...
 
-## `help`
+## `./wallet`
 
-### Parameters
+Runs the wallet without a specified account:
+- If the wallet has only one account, it will be used;
+- If the wallet has more than one account, a selector will be shown to decide which account to use.
+
+The wallet needs to be initialised (`init` command) and with at least one account (`new` command).
```
Comment (Thoralf-M) [inline documentation/docs/02_account_manager.md:18]: Do you think it should be like this? I guess we could also change the `init` to create a new account directly, not sure if that would be too much or better

File: documentation/docs/04_step_by_step.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+> INFO  Created account "savings"
+> CTRL-C
+```
+
+# Send an amount
+
+Get some funds from the faucet.
+```sh
+$ ./wallet main
+> Account "main" command (h for help): faucet https://faucet.alphanet.iotaledger.net/api/enqueue
```
Comment (Thoralf-M) [inline documentation/docs/04_step_by_step.md]: This URL should stay intern

File: documentation/docs/02_account_manager.md (hunk 1 above)
Comment (thibault-martinez) [reply documentation/docs/02_account_manager.md:18]: Tbh I think it's not unreasonable to have them be 2 separate commands

File: documentation/docs/04_step_by_step.md (hunk 2 above)
Comment (thibault-martinez) [reply documentation/docs/04_step_by_step.md]: Removed

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**